test/gcs-test.js

```diff
@@ -0,0 +1,127 @@
+/* eslint-env mocha */
+/* eslint prefer-arrow-callback: "off" */
+
+'use strict';
+
+const crypto = require('bcrypto/lib/random');
+const assert = require('bsert');
+const GCSFilter = require('../lib/golomb/golomb');
+
+const key = crypto.randomBytes(16);
+const P = 20;
+
+const contents1 = [
+  Buffer.from('Alex', 'ascii'),
+  Buffer.from('Bob', 'ascii'),
+  Buffer.from('Charlie', 'ascii'),
+  Buffer.from('Dick', 'ascii'),
+  Buffer.from('Ed', 'ascii'),
+  Buffer.from('Frank', 'ascii'),
+  Buffer.from('George', 'ascii'),
+  Buffer.from('Harry', 'ascii'),
+  Buffer.from('Ilya', 'ascii'),
+  Buffer.from('John', 'ascii'),
+  Buffer.from('Kevin', 'ascii'),
+  Buffer.from('Larry', 'ascii'),
+  Buffer.from('Michael', 'ascii'),
+  Buffer.from('Nate', 'ascii'),
+  Buffer.from('Owen', 'ascii'),
+  Buffer.from('Paul', 'ascii'),
+  Buffer.from('Quentin', 'ascii')
+];
+
+const contents2 = [
+  Buffer.from('Alice', 'ascii'),
+  Buffer.from('Betty', 'ascii'),
+  Buffer.from('Charmaine', 'ascii'),
+  Buffer.from('Donna', 'ascii'),
+  Buffer.from('Edith', 'ascii'),
+  Buffer.from('Faina', 'ascii'),
+  Buffer.from('Georgia', 'ascii'),
+  Buffer.from('Hannah', 'ascii'),
+  Buffer.from('Ilsbeth', 'ascii'),
+  Buffer.from('Jennifer', 'ascii'),
+  Buffer.from('Kayla', 'ascii'),
+  Buffer.from('Lena', 'ascii'),
+  Buffer.from('Michelle', 'ascii'),
+  Buffer.from('Natalie', 'ascii'),
+  Buffer.from('Ophelia', 'ascii'),
+  Buffer.from('Peggy', 'ascii'),
+  Buffer.from('Queenie', 'ascii')
+];
+
+let filter1 = null;
+let filter2 = null;
+let filter3 = null;
+let filter4 = null;
+let filter5 = null;
+
+describe('GCS', function() {
+  it('should test GCS filter build', () => {
+    filter1 = GCSFilter.fromItems(P, key, contents1);
```

Comment:
can't we use the new `BasicFilter` class for these?

Reply:
it would be cool to see some other type of filter get tested in here as well!

Reply:
We could create a new instance of the Golomb class and give it some values other than the Basic filter. This would make a different type of filter and we can then test it.